templates/tensorflow/r2.2/transformer-translate.libsonnet

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local base = import "base.libsonnet";
+local timeouts = import "../../timeouts.libsonnet";
+local tpus = import "../../tpus.libsonnet";
+
+{
+  local transformer = base.GardenTest {
+    modelName: "transformer-translate",
+    command: [
+      "python3",
+      "official/nlp/transformer/transformer_main.py",
+      "--tpu=$(KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS)",
+      "--steps_between_evals=10000",
```

Comment:
If you want, you can set `steps_between_evals` to be the same as the `train_steps` so that the job runs faster

Reply:
Good idea. Done.